main.go

```diff
@@ -312,16 +306,18 @@ func main() {
 			log.Debug("updated cache")
 			if err != nil {
 				log.Error(err)
-				return err
+				return err	
```

Comment:
can we remove the extra space

Reply:
Fixed via https://github.com/crowdsecurity/cs-cloudflare-bouncer/pull/56/commits/8b56fccf8f992071713099b8d387df061530fcee